### PR TITLE
Add the possibility to get segments corresponding to the four quadrant of VLT aperture.

### PR DESCRIPTION
**Pull request by npourre**
Hi! I am a student in astrophysics in Grenoble, France. In a project I am working on, I need to separate the quadrant of the VLT aperture. I have been in touch with Emiel that told me he was interested in merging my solution to get the VLT segmented aperture.

I chose a geometric approach, computing the coordinates of the lines drawn by the spiders and applying conditions (above this line, below this line...) to define the quadrants. Quadrants can be created contiguous or separated by a space corresponding to spiders width (with_segment_gaps=True or False). It works perfectly for my usage but I hope it is suitable for a merging in HCIPy. Feel free to ask me changes! It is my first GitHub branch :D.

Best regards,

Nicolas

### CODE REVIEW

**Comment by ehpor** [comment]
I apparently had an almost-finished PR review pending, but not posted, on this PR. I'll post that review right after this comment for completeness. I still agree with most of those comments, but I don't think it's appropriate nor fair to require you finish the PR after this long of a delay, unless you want to.

@npourre I propose to either
1. merge this PR as is, and I'll take care of the cleanup in a subsequent PR, or
2. keep this PR open, and you do the cleanup before merging.

Let me know what you want to do.

**Review by ehpor — COMMENTED** [review]
Also, testing the segmentation in CI is missing, resulting in the lost test coverage as reported by codecov.

**Comment by npourre** [comment]
> I apparently had an almost-finished PR review pending, but not posted, on this PR. I'll post that review right after this comment for completeness. I still agree with most of those comments, but I don't think it's appropriate nor fair to require you finish the PR after this long of a delay, unless you want to.
> 
> @npourre I propose to either
> 
>     1. merge this PR as is, and I'll take care of the cleanup in a subsequent PR, or
> 
>     2. keep this PR open, and you do the cleanup before merging.
> 
> 
> Let me know what you want to do.

@ehpor , I would be glad to do the cleanup myself. So, keep the PR open and I will deal with it.
I will be on leave for the next three weeks. So I might not make all the changes before September.

**Comment by npourre** [comment]
I cleaned and reshaped the segments computation following your guidelines. It is much shorter indeed!
Feel free to ask me for more changes or to modify the code yourself.

File: hcipy/aperture/realistic.py [hunk 1]
```diff
@@ -22,11 +26,17 @@ def make_vlt_aperture(normalized=False, telescope='ut3', with_spiders=True, with
 		If this is True, a cover will be created for the M3 in stowed position.
 		This M3 cover is only available on UT4, mimicking the ERIS pupil. A warning
 		will be emitted when using an M3 cover with other UTs. Default: False.
+	with_segment_gaps : boolean
```
Comment (ehpor) [inline hcipy/aperture/realistic.py:29]: This argument was removed from the signature. Remove it from the docstring as well.

**Review by ehpor — CHANGES_REQUESTED** [review]
This looks great! Even cleaner than I thought it would be! Thanks!

Only a few things remaining:

- [ ] Rebase your fork and fix the merge conflict. It's a trivial one, but git is choking on it. Let me know if you're unsure on how to do this.
- [ ] Fix the flake8 errors. These are mostly small whitespace issues. See [here](https://github.com/ehpor/hcipy/runs/8102251261) or run flake8 locally.
- [ ] Add a check for the segmentation in the Pytest. This should be as simple as adding `check_segmentation=True` to that function call, similar to other segmentation checks, but I might be mistaken on that.

**Review by ehpor — APPROVED** [review]
LGTM. Thanks for getting this to the finish line. :tada:

**Comment by npourre** [comment]
Great! Thanks a lot Emiel for giving it the last push. :smiley:

**Pull request merged**